### ISSUE TITLE
[mu4e bug] `mu4e-action-add-org-contact` seems broken

### ISSUE THREAD

**Issue by prosoitos** [djcb/mu#2306]
**Describe the bug**

`mu4e-action-add-org-contact` used to work, but since a recent update, that function seems broken.

**How to Reproduce**

After running the function, the proper contact file opens and the sections are correct, but the info doesn't get filed in properly from the email.

Here is what an entry looks like now:

```el
* :name
:PROPERTIES:
:EMAIL: (John Doe :email john.doe@something.com)
:NICK:
:BIRTHDAY:
:ID:       h:22c5becb-2aad-4c99-a1a5-688888888888
:END:
```

Instead of the proper filing:

```el
* John Doe
:PROPERTIES:
:EMAIL: john.doe@something.com
:NICK:
:BIRTHDAY:
:ID:       h:22c5becb-2aad-4c99-a1a5-688888888888
:END:
```

**Environment**
OS: 5.18.14-arch1-1
Emacs: 28.1-7
mu: 1.8.6-1

**Checklist**
- [x] you are running either the latest 1.6.x release, or a 1.8.x release (otherwise, please upgrade)
- [x] you are running mu4e without any third-party extensions (otherwise, make sure you can reproduce without those)
- [x] you have read all of the above

**Comment by djcb** [comment]
Right... mu4e-action-add-org-contact wasn't updated for the new contacts format. I have done so now!
Will be available in 1.8.8 when it comes out (or in the release branch right now).

**Comment by prosoitos** [comment]
Thank you for doing this so quickly!!!! 🙂
